index.html

```diff
@@ -1,52 +1,634 @@
+      <path class="at"
+        d="M1744.84,255.34c-117.51-6.22-235.64,39.19-320.62,120.23a426.65,426.65,0,0,0-55.69,64.92c-18.42,26.45-32.9,55.16-47,84.06-27,55.34-54.36,118.33-108.13,153-55.46,35.78-122.17,22-183.78,33.59-31,5.86-59.82,19.31-87.53,34.05-29.46,15.67-58.38,32.08-89,45.48A682,682,0,0,1,459.34,837c-133.57-24.16-263.19-89-401.55-78.62a359.78,359.78,0,0,0-47,6.72c-1.58.33-.91,2.74.66,2.41,139.29-29.17,274.63,32.92,407.9,63.74,128.36,29.68,262,24.46,386.58-19.3a695.84,695.84,0,0,0,85.35-36.57c30.08-15.34,58.91-33.21,90-46.47,30.85-13.15,62.44-18.53,95.79-20.27,30-1.56,60.35-1.47,89.68-8.57,58.13-14.08,96-59.53,124.1-109.65,31.77-56.73,54.47-117.91,95.19-169.32,75.82-95.71,193.15-156.74,315.16-163.28a409.66,409.66,0,0,1,43.63,0c1.61.08,1.61-2.42,0-2.5Z" />
+      <path class="au"
+        d="M1757,261c-118.93-8.06-238.54,38.27-324.11,120.65a423.34,423.34,0,0,0-56.46,66.86c-17.73,26.07-31.73,54.29-45.37,82.65-27,56.22-54.21,120.51-108.52,155.94-56,36.5-123.46,22.32-185.77,34-31.4,5.87-60.48,19.47-88.52,34.33-29.81,15.81-59.09,32.33-90.1,45.77-125.84,54.56-267.27,70.41-402,44.56-133.8-25.66-263.8-92-402.82-81.59A355.83,355.83,0,0,0,5.84,771c-1.58.34-.91,2.75.66,2.41,140.7-30,277.35,34.4,411.58,66.58C547.44,871,682.85,866.26,808.89,823a694.29,694.29,0,0,0,88-37.17c30.45-15.43,59.62-33.48,91.09-46.86,31.21-13.27,63.14-18.66,96.87-20.35,30.78-1.54,62.07-1.36,92.15-9,58.71-14.83,96.35-61.86,124.19-112.79,30.36-55.55,52.35-115.31,91.14-166,75.72-98.91,195.9-162.69,320.57-168a399.51,399.51,0,0,1,44.18.62c1.6.11,1.6-2.39,0-2.5Z" />
+      <path class="av"
+        d="M1769.23,266.69c-116.89-9.68-234.52,34.72-320.13,114-43.51,40.3-75.31,88.52-101,141.63-24.42,50.43-45.85,105.1-84.38,146.85-19.91,21.58-44.44,38.05-72.74,46.35-34.94,10.24-71.84,8.8-107.82,11-33.79,2.09-65.42,9.23-96.37,23.11-32.6,14.63-63,33.56-95.17,49.11-129.38,62.6-278.34,83.83-420,59.19-140.54-24.44-275.31-98.46-420.89-88.16A350.49,350.49,0,0,0,.9,777c-1.57.34-.91,2.76.66,2.41,146.44-31.94,288.17,39.8,427.75,72.4,136.08,31.8,280,22.22,410.54-27.93,32.07-12.32,62.8-27.36,93-43.74,31.48-17.08,63.27-34.84,98.22-43.75,65.45-16.68,138,2.05,198.11-35.13,45.32-28,72.3-76.73,95.08-123.23,26.06-53.22,49.08-107.32,87-153.55,76.35-93.06,193.24-153,314-156.54a391.73,391.73,0,0,1,44,1.23c1.6.13,1.59-2.37,0-2.5Z" />
+    </svg>
+  </div>
+</body>
+
+</html>
```

Comment:
Missing newline at end of file? You monster.

Reply:
lol